src/views/ComponentsPage.vue

```diff
@@ -49,6 +49,11 @@
         {{ $msTranslate('usage.components.stripe.notInit') }}
       </example-block-line>
     </template>
+    <ms-stripe-card-details
```

Comment:
You can move this in an `<example-block-line>` line 46 next to the other one containing a `ms-stripe-card-details`. Ideally you would also use the `stripeCardDetails` data instead of your static `PAYMENT_STEP_FAKE_DATA` but it doesn't matter much.